api/src/main/java/org/openmrs/module/appointments/notification/impl/DefaultTCAppointmentPatientEmailNotifier.java

```diff
@@ -79,6 +83,33 @@ public NotificationResult sendNotification(final Appointment appointment) throws
         }
     }
 
+    public NotificationResult sendNotification(final Patient patient, final String provider, final String link) throws NotificationException {
+        PersonAttribute patientEmailAttribute = patient.getPerson().getAttribute("email");
+        if (patientEmailAttribute != null) {
+            String patientEmail = patientEmailAttribute.getValue();
+            String patientName = patient.getGivenName();
+            String emailSubject = getAdhocTeleconsultationEmailSubject();
+            String emailBody = getAdhocTeleconsultationEmailBody(patientName, provider, link);
+            String bccEmails = getAdhocTeleconsultationCCEmails();
+            String [] bccEmailArray = null;
+            if (bccEmails != null && !("".equals(bccEmails))) {
+                bccEmailArray = bccEmails.split(",");
+            }
+
+            try {
+                log.info("Sending mail through: " +  mailSender.getClass());
+                mailSender.send(emailSubject, emailBody, new String[] { patientEmail }, null, bccEmailArray);
+                return new NotificationResult("", "EMAIL", 0, EMAIL_SENT);
```

Comment:
Using constants now

Reply:
You could have re-used NotificationResult.SUCCESS_STATUS, instead of creating another local constant. Seems like you may not have realised this already existed.

Reply:
Yeah wasn't aware of that, changed now